devKitLinks.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I would make it start with capital letter as all the other files in this directory.

Reply:
Should it be specified it is APIDevKitLinks and not RenderingDevKit?

Reply:
Renamed to APIDevKitLinks